test/e2e/base/manifests.yaml

```diff
@@ -642,3 +642,150 @@ spec:
+    Z093MEtZcC9xRWlUTWNnWE1hRFM2WWxENzF3U3M3eVhsRUNPUVJYOFIxCkRoTGJnWTU2ejY3U3BP
+    VWpTOFMzenpsbGFUNjV1NnI0bWdRKzFCRUpOWHpoWkd0cDNPa0dqUnk3elhuV3llTXUKTWF2allC
+    eVJZd2NFVDYrS2ZxN2M4dWJLWDIzOWRiZy9rckNUMmNPa0NFdjR6Qm12b0NzRnBLVXRlamwxczlr
+    MQo2M2xwTjhNSmU3cDZ6QXJ4eTB3dndSdEdHR0pGc0VuUnM2Q2JBOUZ3ZndLQmdRRGV5a1NkdWdU
+    TEtvdWRuMVdSCnNXbkV0cy9WaGxySENPaUtJL2ZiaWZMTmQyaDkxM3dMSlVXcTN1eVVhSk5LelVY
+    QzhqdFE2ZVZFVGZrSmduZDEKY1Y1SkFZc1hDTm9WRnduaXhDZ0U3NkpxaW5WU3gxNnhIWXJxUlNH
+    R2pkMGxmRmNBM1F4anNWUWgyQ1VtUjJSWQp0WmMvSkdDQVdRN0VPR2xIblBVcUo0Smo1d0tCZ1FE
+    WWlmV0hodEdrOUd5VlRhM3lUdmlldkUyNzlOb0wzMU84ClZxTjArWjRIdnVwa25UNU1BS1luOTBD
+    eDZscjg0Q0VkY1dSRGpGeEJnNXVDUm5waHRSaUNDcU1kc0dKSC94bUsKWXlhMDRBMFlUOURaMmk4
+    T3FYazNGWlFJZVJhT3h2clZFSlhvbHRRSENZbzRoRGFjTkQ4MHdwMmVoc28rZ3loeQp5M3pVYUFT
+    dGF3S0JnUUNub0l5c0dEQlBKUDNUWlBVZkg5dXk5TkcvbU0xb2dCMW5BdFVVeStCeUUrdjg4MHNU
+    Cm44R2haUVdnVEdhMm0yaG4yNVlodFRxWmdBbG16RkZSZmpFd3dRYUVHRmJ1RTEyT3o0UkU0b0RU
+    cE1YTnVOQlkKZHgvMFVIdC9IL0pLcUJsV1B2dDVidzBjUWVNQndDbVNUVjgxVlF0K3hvTkR4Q1lD
+    aWVnMUJYRW45UUtCZ1FDeApESFFONE9Udkc4N2djTWdtRS8zaVJCTEcrSmVFRjVEdXNHUXIwNkVy
+    c2lVU0tTTnpueUE4RTBUNWM2WDdsd0pjCmdZYzZWN3hpOE1tb3hobHdaRkxxbU9udjczVWNhdTht
+    SUpYWVZFMUFzSm9TVmZ3ZDMrRzFjVlk5YTB4UFoxQlYKLzU2aXN4Z2Z2T1FmKzdKdkc2VUdiWTNV
+    dlY4RnA5WGRycFdkVW9Ob2l3S0JnUURlQjgvRzhMRyt4MlEwYUFkWgpteHFrQTQzYUtoQ01WZjFh
+    WVF0S3cwVDU0czNkdUNDTkY1cGJQeHpZWTFUeVNRS0ljS1hFZlBiRWhFdUZOTXpICnVRV0ZOT1Zh
+    VjRST25lWkNIVGFZbTRFekQ5SWJVRktEZXlpVW0zZXVaRG85dm16VldZd2ZyVUQwOXc3ZFRKTWYK
+    MmExcG5RVm84bVlCaFFCdFI2Tk9jTmtFQlE9PQotLS0tLUVORCBQUklWQVRFIEtFWS0tLS0tCg==
+---
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: http3-gateway
+  namespace: gateway-conformance-https
+spec:
+  gatewayClassName: "{GATEWAY_CLASS_NAME}"
+  listeners:
+  - name: https
+    protocol: HTTPS
+    port: 443
+    tls:
+      mode: Terminate
+      certificateRefs:
+      - kind: Secret
+        name: http3-secret
+        namespace: gateway-conformance-https
+  - name: http
+    protocol: HTTP
+    port: 80
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: http3-backend
+  namespace: gateway-conformance-https
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: http3-backend
+  namespace: gateway-conformance-https
+  labels:
+    service: http3-backend
+    app: http3-backend
+spec:
+  ports:
+  - name: http3
+    port: 3000
+    targetPort: 3000
+  selector:
+    app: http3-backend
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: http3-backend
+  namespace: gateway-conformance-https
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: http3-backend
+      version: v1
+  template:
+    metadata:
+      labels:
+        app: http3-backend
+        version: v1
+    spec:
+      serviceAccountName: http3-backend
+      containers:
+      - image: gcr.io/k8s-staging-ingressconformance/echoserver:v20221109-7ee2f3e
```

Comment:
`gcr.io/k8s-staging-gateway-api/echo-basic:v20231214-v1.0.0-140-gf544a46e`

We are using this as default demo server.